src/datasets.cpp

```diff
@@ -39,409 +39,352 @@
-                "\"9\" indicates a User Defined Priority. The numeral \"0\" is reserved "
-                "for future use."),
-                false, false, 1, 1, Exiv2::string, IptcDataSets::envelope, ""),
-        DataSet(IptcDataSets::DateSent, "DateSent", N_("Date Sent"),
-                N_("Uses the format CCYYMMDD (century, year, month, day) as de-fined "
-                "in ISO 8601 to indicate year, month and day the service sent the material."),
-                true, false, 8, 8, Exiv2::date, IptcDataSets::envelope, ""),
-        DataSet(IptcDataSets::TimeSent, "TimeSent", N_("Time Sent"),
-                N_("Uses the format HHMMSS:HHMM where HHMMSS refers to "
-                "local hour, minute and seconds and HHMM refers to hours and "
-                "minutes ahead (+) or behind (-) Universal Coordinated Time as "
-                "described in ISO 8601. This is the time the service sent the material."),
-                false, false, 11, 11, Exiv2::time, IptcDataSets::envelope, ""),
-        DataSet(IptcDataSets::CharacterSet, "CharacterSet", N_("Character Set"),
-                N_("This tag consisting of one or more control functions used for the announcement, "
-                "invocation or designation of coded character sets. The control functions follow "
-                "the ISO 2022 standard and may consist of the escape control "
-                "character and one or more graphic characters."),
-                false, false, 0, 32, Exiv2::string, IptcDataSets::envelope, ""),
-        DataSet(IptcDataSets::UNO, "UNO", N_("Unique Name Object"),
-                N_("This tag provide a globally unique "
-                "identification for objects as specified in the IIM, independent of "
-                "provider and for any media form. The provider must ensure the "
-                "UNO is unique. Objects with the same UNO are identical."),
-                false, false, 14, 80, Exiv2::string, IptcDataSets::envelope, ""),
-        DataSet(IptcDataSets::ARMId, "ARMId", N_("ARM Identifier"),
-                N_("The DataSet identifies the Abstract Relationship Method identifier (ARM) "
-                "which is described in a document registered by the originator of "
-                "the ARM with the IPTC and NAA organizations."),
-                false, false, 2, 2, Exiv2::unsignedShort, IptcDataSets::envelope, ""),
-        DataSet(IptcDataSets::ARMVersion, "ARMVersion", N_("ARM Version"),
-                N_("This tag consisting of a binary number representing the particular "
-                "version of the ARM specified by tag <ARMId>."),
-                false, false, 2, 2, Exiv2::unsignedShort, IptcDataSets::envelope, ""),
-        DataSet(0xffff, "(Invalid)", "(Invalid)",
-                "(Invalid)", false, false, 0, 0, Exiv2::unsignedShort, IptcDataSets::envelope, "")
+    static constexpr DataSet envelopeRecord[] = {
+        {IptcDataSets::ModelVersion, "ModelVersion", N_("Model Version"),
```

Comment:
Could you please revert the formatting changes here and just keep the 1-line change about `static constexpr` ?